src/monkey_patch/function_modeler.py

```diff
@@ -370,21 +363,23 @@ def _execute_finetuning(self, function_description, func_hash):
             return
 
         # here can be sure that datasets were read in as that is checked in the finetune_check
-        align_dataset_size = self.dataset_sizes["alignments"][func_hash] if func_hash in self.dataset_sizes["alignments"] else 0
-        patch_dataset_size = self.dataset_sizes["patches"][func_hash] if func_hash in self.dataset_sizes["patches"] else 0
+        align_dataset_size = self.dataset_sizes["alignments"][func_hash] if func_hash in self.dataset_sizes[
+            "alignments"] else 0
+        patch_dataset_size = self.dataset_sizes["patches"][func_hash] if func_hash in self.dataset_sizes[
+            "patches"] else 0
         total_dataset_size = align_dataset_size + patch_dataset_size
         training_file_id = response["id"]
         # submit the finetuning job
         try:
             finetuning_response = openai.FineTuningJob.create(training_file=training_file_id, model="gpt-3.5-turbo",
```

Comment:
We need to figure out a way to call the logic to trigger a finetune in the relevant LLM provider class (i.e OpenAI, Bedrock, etc) - this is not the right place for this logic to live now we are adding support for more providers.